welcome/learn_more_about_openshift.adoc

```diff
@@ -118,7 +118,7 @@ Use the following sections to find content to help you learn about and use {prod
 | link:https://developers.redhat.com/[Red Hat Developers site]
 | xref:../cicd/builds/understanding-image-builds.adoc#understanding-image-builds[Builds]
 
-| link:https://developers.redhat.com/products/codeready-workspaces/overview[Red Hat CodeReady Workspaces]
+| link:https://developers.redhat.com/products/openshift-dev-spaces/overview[{openshift-dev-spaces-productname}]
```

Comment:
The full product name is ` Red Hat OpenShift Dev Spaces`.

In which file do you keep the value for the `{openshift-dev-spaces-productname}` attribute?